internal/indexer/definitions/funfile.yaml

```diff
@@ -0,0 +1,76 @@
+---
+#id: funfile
+name: FunFile
+identifier: funfile
+description: Welcome to funfile.org, where files are fun!
+language: en-us
+urls:
+  - https://funfile.org/
```

Comment:
Should this be changed to https://www.funfile.org/ since they redirect to that? The announces use https://www.funfile.org/ as well.

Reply:
Yeah that is probably wise to change